app/components/ch/components/DownloadList.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <li>
+    <button
+      class="download-item"
+      :href="url"
+      target="_blank"
+      :download="title"
+      :id="`download-item-${id}`"
+    >
+      <SvgIcon icon="Download" size="xl" class="download-item-icon" />
+      <div>
+        <p class="download-item-title">
```

Comment:
We use pure old BEM nomenclature in the project, so please rename your classes like `download-item__title` in place of `download-item-title`